packages/react/src/TreeView/TreeView.tsx

```diff
@@ -488,6 +511,12 @@ const Item = React.forwardRef<HTMLElement, TreeViewItemProps>(
             <div style={{gridArea: 'spacer', display: 'flex'}}>
               <LevelIndicatorLines level={level} />
             </div>
+            {dragHandle ? (
+              // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+              <div className="PRIVATE_TreeView-item-drag-handle" onMouseDown={() => setIsExpanded(false)}>
+                {dragHandle}
```

Comment:
I don't think we should require folks to style their own drag handles. We should ship with a default recommended style.